src/main/io/serial_cli.c

```diff
@@ -970,6 +973,61 @@ static void cliSerial(char *cmdline)
 
 }
 
+static void cliSerialPassthrough(char *cmdline)
+{
+    if (isEmpty(cmdline)) {
+        cliShowParseError();
+        return;
+    }
+
+    int id = -1;
+    unsigned baud = 0;
+    char* tok = strtok(cmdline, " ");
+    int index = 0;
+
+    while (tok != NULL) {
+        switch(index) {
+            case 0:
+                id = atoi(tok);
+                break;
+            case 1:
+                baud = atoi(tok);
+                break;
+        }
+        index++;
+        tok = strtok(NULL, " ");
+    }
+
+    serialPort_t *passThroughPort;
+    serialPortUsage_t *passThroughPortUsage = findSerialPortUsageByIdentifier(id);
+    if (!passThroughPortUsage || passThroughPortUsage->serialPort == NULL) {
+        if (!baud)
+        {
+            printf("Port %d is not open, you must specify baud\r\n", id);
+            return;
+        }
+
+        passThroughPort = openSerialPort(id, FUNCTION_PASSTHROUGH, NULL,
+                                         baudRates[baud], MODE_RXTX,
+                                         SERIAL_NOT_INVERTED);
+        if (!passThroughPort) {
+            printf("Port %d could not be opened\r\n", id);
+            return;
+        }
+        printf("Port %d opened, baud=%d\r\n", id, baudRates[baud]);
+    } else {
+        passThroughPort = passThroughPortUsage->serialPort;
+        if (!(passThroughPort->mode & MODE_TX))
```

Comment:
`serialRxBytesWaiting()` must return 0/false if port is TX only (or it is buggy).
 `serialWrite()` should handle RX only port without breaking something else.
IMO it is not necessary to handle this explicitly in `serialPassThrough`, the overhead is negligible.

It is safer to keep RX/TX port mode unchanged when port is already open and let user override this.
When opening port, use RXTX by default, but allow override.

Another modes may be interesting in future (inversion/singlewire), but that is usable mostly only for developers